scipy/special/_generate_pyx.py

```diff
@@ -851,7 +852,8 @@ def _get_conditional(self, types, codes, adverb):
             if len(typcode) == 1:
                 continue
             if typ not in seen:
-                clauses.append("{} is {}".format(typ, underscore(CY_TYPES[code])))
+                clauses.append("{} is {}".format(typ,
```

Comment:
I guess this change is made for line length? perhaps an f-string?
`clauses.append(f"{typ} is {underscore(CY_TYPES[code])}")`

Reply:
The current change here is purely a style change, and it makes the code read worse rather than better.

Reply:
ok